index.html

```diff
@@ -448,6 +630,48 @@ <h2 id="deviceId">
         </p>
       </section>
     </section>
+    <section>
+      <h1>Privacy Indicator Requirements</h1>
+      <p>This specification extends the <a href=
+        "https://w3c.github.io/mediacapture-main/#privacy-indicator-requirements">
+        Privacy Indicator Requirements</a> of
+      <code><a>getUserMedia</a></code> to include <code><a>getDisplayMedia</a></code>.</p>
+      <p>References in this specification to [[\devicesLiveMap]],
+      [[\devicesAccessibleMap]], and [[\kindsAccessibleMap]] refer to the
+      definitions already created to support Privacy Indicator Requirements for
+      <code><a>getUserMedia</a></code>.</p>
+      <p>For each <var>kind</var> of device that
+      <code><a>getDisplayMedia</a></code> exposes, set <var>kind</var>
+      to the permission name associated with <var>kind</var> for obtaining
+      display devices (e.g. "screen" for video), and do the following:
+      <ul>
+        <li>Define <var>any&lt;kind&gt;Accessible</var> (e.g.
+          <var>anyScreenAccessible</var>) as the
+          logical OR of the [[\kindsAccessibleMap]]<var>[kind]</var> value and all
+          the [[\devicesAccessibleMap]]<var>[deviceId]</var> values for devices of
+          that kind, and initialize all values to <code>false</code>.
+        </li>
+        <li>Define any<var>&lt;kind&gt;Live</var> (e.g. <var>anyScreenLive</var>)
+          to be the logical OR of all the
+          [[\devicesLiveMap]]<var>[deviceId]</var> values for devices of that
+          kind.
+        </li>
+      </ul>
+      <p>Then, given the new definitions above, the requirements on the User
+      Agent are those specified in <a href=
+        "https://w3c.github.io/mediacapture-main/#privacy-indicator-requirements">
+        Privacy Indicator Requirements</a> of
+      <code><a>getUserMedia</a></code>.</p>
+      <div class="note">
+        <p>
+        Since this specification forbids user agents from persisting "granted"
+        permissions, only the "Live" indicators are significant.
+        </p>
+      </div>
+      <p>The User Agent MUST NOT implement the <code><a href=
```

Comment:
"MUST NOT implement" sounds a bit strange. Would it be clearer to say "MUST NOT fire the devicechange event based on changes in the set of available sources for getDisplayMedia"?